utils/programs/assetround.teal

```diff
@@ -0,0 +1,120 @@
+//
```

Comment:
An overview similar to utils/programs/globcheck.teal  with be super helpful here.